src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -227,8 +227,8 @@ def load_arguments(self, _):
             c.argument('settings', nargs='+', help="space-separated app settings in a format of `<name>=<value>`")
             c.argument('setting_names', nargs='+', help="space-separated app setting names")
         with self.argument_context(scope + ' config ssl import') as c:
-            c.argument('key_vault', help='The name or resource ID of the Key Vault')
-            c.argument('key_vault_certificate_name', help='The name of the certificate in Key Vault')
```

Comment:
Don't remove this at this time. Keep it as is to avoid breaking changes.

Make the Certificate_name new parameter and update the help to say name of the certificate in the keyvalut to be imported or name of the app service certificate. For the old parameter follow the breaking change protocol to add an attribute which states that this will be deprecated in future releases and should be updated to use the certificate_name parameter.

Reply:
I need to chose a release version to put into the documentation, what version do we want?

Reply:
what documentation you are referring to? the release documentation gets auto generated with the title of the PR

Reply:
discussed offline, updated deprecation version to 2.15.0